src/main/webapp/intrigue-api/metacards/metacards.js

```diff
@@ -433,6 +447,8 @@ const resolvers = {
     createMetacard,
     saveMetacard,
     deleteMetacard,
+    subscribeToWorkspace,
+    unsubscribeFromWorkspace,
```

Comment:
I think we should rename these to be a little more generic
```suggestion
    subscribeToMetacard,
    unsubscribeFromMetacard,
```

Reply:
Well this resolver only hits those endpoints. And their sole job is to subscribe and unsubscribe to/from workspaces. Do you mean having one resolver that handles both? Because idk how to generalize it more than that.

Reply:
Sorry, I didn't realize the endpoint was coupled to workspaces